extra_tests/snippets/vm_frame_locals.py

```
@@ -0,0 +1,255 @@
+            frame.f_locals["extra"] = ReentrantFinalizer(frame, "extra")
+            return frame
+
+        frame = frame_with_extra()
+        frame.clear()
+        self.assertEqual(events, [None])
+
+        events.clear()
+
+        def frame_with_overwritten_fast_local():
+            frame = sys._getframe()
+            value = ReentrantFinalizer(frame, "value")
```

Flake8 (7.3.0)

[error] local variable 'value' is assigned to but never used

(F841)